src/react/src/components/DashboardClaimsListTableHeader.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useCallback } from 'react';
+import { func, string } from 'prop-types';
+import TableHead from '@material-ui/core/TableHead';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+
+const DashboardClaimsListTableHeaderStyles = Object.freeze({
```

Comment:
As it a new changes, we could also use classes here instead of const.

Reply:
Done